src/charts/line/line-chart.directive.js

```diff
@@ -0,0 +1,287 @@
+/**
+ * @ngdoc directive
+ * @name patternfly.charts.directive:pfLineChart
+ *
+ * @description
+ *   Directive for rendering a line chart.
+ *   <br><br>
+ *   See http://c3js.org/reference.html for a full list of C3 chart options.
+ *
+ * @param {object} config configuration settings for the line chart:<br/>
+ * <ul style='list-style-type: none'>
+ * <li>.chartId            - the ID of the container that the chart should bind to
+ * <li>.units              - unit label for values, ex: 'MHz','GB', etc..
+ * <li>.tooltipType        - (optional) set the type of tooltip, valid values:
+ * <ul style='list-style-type: none'>
+ * <li>'default'           - show the data point value and the data point name.
+ * <li>'valuePerDay'       - show the date and value for the data point.
+ * </ul>
```

Comment:
Does this tooltip type make sense for a line chart wrapper? I believe these come from the sparkline chart wrapper which has a more specific purpose.